src/Fhir.Anonymizer.Core.UnitTests/ExtensionTests/ElementNodeNavExtensionTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
```

Comment:
Could we place all tests for extensions in the same folder? Currently it seems some are placed in Extensions, while some are placed in ExtensionTests.

Reply:
Sure, merge them to same place